pkg/pillar/types/zedroutertypes.go

```diff
@@ -792,6 +816,42 @@ const (
 	MST_LAST = 255
 )
 
+// ServerProbe - remote probe info configured from the cloud
+type ServerProbe struct {
+	ServerURL     string // include method,host,paths
+	ServerIP      net.IP
+	ProbeInterval uint32 // probe frequence in seconds
+}
+
+// ProbeInfo - per phyical port probing info
+type ProbeInfo struct {
+	IfName    string
+	IsPresent bool
+	// local nexthop probe state
+	GatewayUP  bool
+	LocalAddr  net.IP
+	NhAddr     net.IP
+	IsFree     bool
+	FailedCnt  uint32
+	SuccessCnt uint32
```

Comment:
Are these consequtive successes with no intervening failures? Should be commented.